src/localization/bundles/en.json

```diff
@@ -150,7 +150,7 @@
     "validate.config.sshPassphrase.missing": "An SSH passphrase is required in the Deployment tab.",
     "validate.deployment.doWipeDisks": "Wipe data disks",
     "validate.deployment.doWipeDisks.tooltip": "Wipes the data and destroys the GPT and MBR on each non-system disk before cloud deployment.",
-    "validate.deployment.doWipeDisks.warning": "Warning: all data disks on selected servers will be wiped.",
+    "validate.deployment.doWipeDisks.warning": "Warning: all data disks on selected servers will be wiped. Wiping disk can only be performed once for a server during the cloud deployment. If the server's OS was installed manually, please make sure it can be accessed by ssh without passsword.",
```

Comment:
The second sentence should begin with "Wiping disks" rather than "Wiping disk".

Reply:
Additionally should change the very end to:
> without **a** password.

Could also change "make sure" to "ensure" but that is totally up to taste.